x-pack/plugins/profiling/server/lib/setup/get_setup_instructions.ts

```diff
@@ -23,29 +22,21 @@ export interface SetupDataCollectionInstructions {
 export async function getSetupInstructions({
   packagePolicyClient,
   soClient,
+  apmServerHost,
 }: {
   packagePolicyClient: PackagePolicyClient;
   soClient: SavedObjectsClientContract;
+  apmServerHost?: string;
 }): Promise<SetupDataCollectionInstructions> {
-  const [collectorPolicy, apmPolicy] = await Promise.all([
-    getCollectorPolicy({ packagePolicyClient, soClient }),
-    getApmPolicy({ packagePolicyClient, soClient }),
-  ]);
+  const collectorPolicy = await getCollectorPolicy({ packagePolicyClient, soClient });
 
   if (!collectorPolicy) {
     throw new Error('Could not find Collector policy');
   }
 
-  if (!apmPolicy) {
-    throw new Error('Could not find APM policy');
-  }
-
   const collectorVars = collectorPolicy.inputs[0].vars;
-  const apmServerVars = apmPolicy.inputs[0].vars;
-
-  const apmHost: string | undefined = apmServerVars?.host?.value;
-  const symbolizerHost = apmHost?.replace(/\.apm\./, '.symbols.');
-  const collectorHost = apmHost?.replace(/\.apm\./, '.profiling.')?.replace('https://', '');
+  const symbolizerHost = apmServerHost?.replace(/\.apm\./, '.symbols.');
+  const collectorHost = apmServerHost?.replace(/\.apm\./, '.profiling.')?.replace('https://', '');
```

Comment:
Is this working as intended?
Right now i see the component name in the domain is being stripped completely